librz/core/core.c

```diff
@@ -2502,12 +2508,17 @@ RZ_API bool rz_core_init(RzCore *core) {
 	rz_core_cmd_init(core);
 	rz_core_plugin_init(core);
 
-	core->dbg = rz_debug_new(true);
+	RzBreakpointContext bp_ctx = {
+		.user = core,
+		.is_mapped = bp_is_mapped,
+		.maps_sync = bp_maps_sync,
+		.bits_at = bp_bits_at
```

Comment:
be sure that windows is ok with this. i suggest t convert it to single assignment and have the struct initialized to 0.

Reply:
It is only a problem with c++